networkx/algorithms/tests/test_dag.py

```diff
@@ -270,6 +270,10 @@ def test_ancestors(self):
         assert ancestors(G, 1) == set()
         pytest.raises(nx.NetworkXError, ancestors, G, 8)
 
+    def test_ancestors_not_implemented(self):
```

Comment:
```suggestion
    def test_ancestors_undirected(self):
```
Very minor nit, but a perhaps slightly more meaningful name for the test. Just a suggestion, feel free to ignore - but if you do accept it be sure to change the name for `test_descendants_not_implemented` too so that they're consistent!

Reply:
I fully agree with such names